src/hooks/use-slides.js

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import { ulid } from 'ulid';
 
-const PLACEHOLDER_CLASS_NAME = 'slide-placeholder';
+export const PLACEHOLDER_CLASS_NAME = 'spectacle-v7-slide';
```

Comment:
Do we need a TODO/ticket to replace this before merge to main?

Reply:
`PLACEHOLDER_CLASS_NAME` is probably not the right variable name. It's essentially the class selector we apply to every `<Slide />` `div` no matter the origin, JSX, MDX, MD, SlideSet. Maybe `SPECTACLE_SLIDE_SELECTOR`, I'm unsure if it should be a randomly generated ID (that would be consistent through the lifecycle of the deck) or just a string.

Reply:
I'm ok with variable name, I meant `spectacle-v7-slide`. Are we intending to change this every semver major for some reason?

Reply:
But go ahead with var name change if you think that's better!